MAINTAINERS.md

```diff
@@ -23,6 +23,7 @@
 | Lucas Saldanha   | lucassaldanha    | lucassaldanha    |
 | Sally MacFarlane | macfarla         | macfarla         |
 | Karim Taam       | matkt            | matkt            |
+| Matilda Clerke   | Matilda-Clerke   | MatildaClerke    |
```

Comment:
nit: alphabetical sort means it should be one line above

Reply:
Thanks